lib/metadataTypes/DataExtension.js

```diff
@@ -425,8 +426,7 @@ class DataExtension extends MetadataType {
             cache.mergeMetadata('folder', result.metadata, this.buObject.eid);
 
             // get the types and clean out non-shared ones
-            const folderTypesFromParent = require('../MetadataTypeDefinitions').folder
-                .folderTypesFromParent;
+            const folderTypesFromParent = MetadataTypeDefinitions.folder.folderTypesFromParent;
```

Comment:
we are not importing `MetadataTypeDefinitions` here? This should throw an error

Reply:
MetadataTypeDefinitions is imported elsewhere. We need to tidy this up longer term, but in the interests of changing as little as possible, i didnt yet - see line 1117